src/components/Documentation/RightPanel/index.tsx

```diff
@@ -98,16 +107,16 @@ const RightPanel: React.FC<IRightPanelProps> = ({
           </>
         )}
         {headings.map(({ slug, text }) => (
-          <Link
+          <GatsbyLink
             className={cn(
               styles.headingLink,
               current === slug && styles.current
             )}
             key={`link-${slug}`}
-            href={`#${slug}`}
+            to={`${window.location.pathname}#${slug}`}
```

Comment:
This isn't building:
```
...
       failed Building static HTML for pages - 18.238s
error "window" is not available during server side rendering.
       
         114 |             )}
         115 |             key={`link-${slug}`}
       > 116 |             to={`${window.location.pathname}#${slug}`}
             |                    ^
         117 |           >
         118 |             {text}
         119 |           </GatsbyLink>
       
       
         WebpackError: ReferenceError: window is not defined
         
         - index.tsx:116 
           src/components/Documentation/RightPanel/index.tsx:116:20
         
         - index.tsx:109 RightPanel
           src/components/Documentation/RightPanel/index.tsx:109:19
         
       
       not finished Generating image thumbnails - 719.726s
error Command failed with exit code 1.
       info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
------------------------
Error: Command failed: yarn build
    at checkExecSyncError (child_process.js:611:11)
    at execSync (child_process.js:647:15)
    at run (/tmp/build_654cf463aa916749b29d4c6b1456d24e/scripts/deploy-with-s3.js:31:3)
    at main (/tmp/build_654cf463aa916749b29d4c6b1456d24e/scripts/deploy-with-s3.js:106:5) {
  status: 1,
  signal: null,
...
```
Please test your changes locally. See https://dvc.org/doc/user-guide/contributing/docs

Reply:
Thanks! fixed.